alphagov/paas-cf
# [#150881142] Update pipeline variables to new format

## What

We've upgraded concourse to the version 3.4.1 not so long ago. Although
the old templating is still supported it will be deprecated in the
future.

The new format supports proper types and variable interpolation.

Pipecleaner had to be modified in order to fit the new syntax.

Some of the functionality has been fixed after the upgrade. Removing it
may benefit us with a clean codebase.

## How to review

- Sanity code check
- See if there aren't variables left out
- See if related `FIXME`s have been resolved

I have tested pushing the pipelines and running the `create-cloudfoundry` pipeline.
